toolkit/tools/pkg/imagecustomizerlib/testdata/verity-usr-config.yaml

```diff
@@ -0,0 +1,83 @@
+storage:
+  bootType: efi
+  disks:
+  - partitionTableType: gpt
+    maxSize: 5120M
+    partitions:
+    - id: esp
+      type: esp
+      start: 1M
```

Comment:
I would use `size` for partitions and omit `maxSize`. This should make the config a little simpler.